test/unit/SwapOperator/swapETHForEnzymeVaultShare.js

```diff
@@ -0,0 +1,346 @@
+const { ethers } = require('hardhat');
+const { expect } = require('chai');
+const { time } = require('@openzeppelin/test-helpers');
+const { BigNumber } = require('ethers');
+const { ETH } = require('../../../lib/constants').Assets;
+const { hex } = require('../utils').helpers;
+
+const {
+  utils: { parseEther },
+} = ethers;
```

Comment:
nit: this fits on 1 line

Reply:
the linter wants this.

breaks the build if i line it up.

We can look into the linting later, i'd merge this as is since it's a nit.